home/management/commands/update_show_in_menu.py

```diff
@@ -0,0 +1,8 @@
+from django.core.management.base import BaseCommand
+
+from wagtail.core.models import Page
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        Page.objects.all().update(show_in_menus=True)
```

Comment:
I don't think its a good idea to update all `Page`s. I think it would be safer to limit updates to `Page`s where `show_in_menus` is true by default.

Reply:
Fixed it!